packages/pluggableWidgets/datagrid-web/src/components/Table.tsx

```diff
@@ -60,6 +64,9 @@ export function Table<T>(props: TableProps<T>): ReactElement {
     const [paginationIndex, setPaginationIndex] = useState<number>(0);
     const [sortBy, setSortBy] = useState<Array<SortingRule<object>>>([]);
     const [filters, setFilters] = useState<Filters<object>>([]);
+    const [columnsWidth, setColumnsWidth] = useState<ColumnWidth>(
+        props.columns.map((_c, index) => ({ [index.toString()]: undefined })).reduce((p, c) => ({ ...p, ...c }), {})
```

Comment:
This got me thinking, would this be easier if we used es2017 features? Can we use those fearures in pluggable-widgets?

Reply:
We are maybe in sync then, because I was thinking.. maybe I should use flatMap, then I’ve changed to es2019 to test and it was working fine but of course flatMap is not the right function to use here :)

Reply:
Isn't it `Object.fromEntries`?

Reply:
I changed general TSConfig from PIW-Tools to ES2019 (lib) and applied `Object.fromEntries` instead

Reply:
✅